diff --git a/node_modules/@react-native-oh-tpl/react-native-view-shot/src/NativeModule.js b/node_modules/@react-native-oh-tpl/react-native-view-shot/src/NativeModule.js
index 8e45f94..fc40b12 100644
--- a/node_modules/@react-native-oh-tpl/react-native-view-shot/src/NativeModule.js
+++ b/node_modules/@react-native-oh-tpl/react-native-view-shot/src/NativeModule.js
@@ -1,13 +1,11 @@
 // @flow
-import { NativeModules } from 'react-native';
+import { NativeModules, Platform } from 'react-native';
 
-// @ts-ignore
-const isTurboModuleEnabled = global.__turboModuleProxy != null;
+const isTurboModuleEnabled = global.__turboModuleProxy != null || (Platform.OS !== 'android' && Platform.OS !== 'ios');
 
 const RNViewShot = isTurboModuleEnabled
-  ? // eslint-disable-next-line @typescript-eslint/no-var-requires
-    require('./NativeViewShot').default
+  ? require('./NativeViewShot').default
   : NativeModules.RNViewShot;
 
 export default RNViewShot;
