diff --git a/android/CMakeLists.txt b/android/CMakeLists.txt
index 8ba41221e81fc2b09215f91870ec2f1060e9fd03..799f21f480c7dd7fb98e0433db751f9f5f3d346b 100644
--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -22,6 +22,9 @@ string(APPEND CMAKE_CXX_FLAGS " -DREACT_NATIVE_MINOR_VERSION=${REACT_NATIVE_MINO
 
string(APPEND CMAKE_CXX_FLAGS " -fexceptions -fno-omit-frame-pointer -frtti -fstack-protector-all -std=c++${CMAKE_CXX_STANDARD} -Wall -Werror")

# Needed for newer NDK/Clang where VLA in C++ triggers -Wvla-cxx-extension and is promoted to error via -Werror.
string(APPEND CMAKE_CXX_FLAGS " -Wno-vla-extension")
+
 if(${IS_NEW_ARCHITECTURE_ENABLED})
     string(APPEND CMAKE_CXX_FLAGS " -DRCT_NEW_ARCH_ENABLED")
 endif()
