diff --git a/android/CMakeLists.txt b/android/CMakeLists.txt
--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -19,7 +19,8 @@ endif()

 string(APPEND CMAKE_CXX_FLAGS " -DREACT_NATIVE_MINOR_VERSION=${REACT_NATIVE_MINOR_VERSION} -DREANIMATED_VERSION=${REANIMATED_VERSION} -DHERMES_ENABLE_DEBUGGER=${HERMES_ENABLE_DEBUGGER}")

string(APPEND CMAKE_CXX_FLAGS " -fexceptions -fno-omit-frame-pointer -frtti -fstack-protector-all -std=c++${CMAKE_CXX_STANDARD} -Wall -Werror")
string(APPEND CMAKE_CXX_FLAGS " -Wno-vla-extension")

